### DISCUSSION TITLE
When new version of gunicorn will release?

### DISCUSSION THREAD

**Discussion by JugeshHM** [benoitc/gunicorn#2989] [General]
We are actively using gunicorn in our python-flask application. The latest gunicorn 20.1.0 version was released on Mar 27, 2021.
Can you please tell me is there any plan of releasing the new version of gunicorn in near future?
TIA

**Comment by benoitc** [comment]
this has already been answered. Track the 21.0 milestone. While I hear the frustration, the release will happen when this will be closed.
